modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/cards/Cards.tsx

```diff
@@ -56,6 +61,37 @@ const Card = ({item, schema}: {item: any; schema: ICardSchema}) => {
 		actionsRef.current &&
 		(filterItemActions(actionsRef.current, item) as any);
 
+	const getLabels = (
+		item: any
+	): Array<{
+		displayType: TDisplayType;
+		value: string;
+	}> => {
+		if (!schema.labels || !Array.isArray(schema.labels)) {
```

Comment:
Any special reason for `isArray` check? It's not wrong, just seems unnecessary.

Reply:
Leftover from when I was tinkering with things.